src/service/userDetails.js

```diff
@@ -436,7 +446,7 @@ class UserDetails {
         })
         .then((t) => {
           transaction = t;
-          return UserDetails._createTokenPair(userId, persona, transaction);
+          return UserDetails._createTokenPair(userId, persona, platform);
```

Comment:
Looks like you have remove the `transaction` - is that intentional? If so, then why assign to the variable in the previous line?

Reply:
Thanks @vraravam, my bad. I overlooked the function parameters and removed it considering it was not necessary. It is fixed now.